README.md

```diff
@@ -16,3 +16,9 @@ If you want to have system site packages, use
 ```
     virtualenv_site_packages: "--system-site-packages"
 ```
+
+If you want to specify virtualenv version, use
+
+```
+  virtualenv_version : "13.0.0"
+```
```

Comment:
This should probably have a quick note about the latest version being installed if a specific one is not configured by the role user.